src/screens/Profiles/UserLibrary/components/UserLibraryListScreen/component.js

```diff
@@ -128,52 +167,40 @@ export class UserLibraryListScreenComponent extends React.Component {
 
   renderFooter = () => {
     const { loading } = this.props;
-    if (!loading) return (<View />);
-
+    if (!loading) return <View />;
     return (
       <ActivityIndicator color="white" style={{ paddingVertical: 16 }} />
     );
   }
 
-  renderItem = ({ item }) => (
+  renderRow = (_type, data) => (
     <UserLibraryListCard
       currentUser={this.props.currentUser}
-      libraryEntry={item}
+      libraryEntry={data}
       libraryStatus={this.props.libraryStatus}
       libraryType={this.props.libraryType}
       navigate={this.props.navigation.navigate}
       profile={this.props.navigation.state.params.profile}
-      updateUserLibraryEntry={this.updateUserLibraryEntry}
-      onSwipingItem={this.onSwipingItem}
+      updateUserLibraryEntry={this.updateUserLibraryEntry.bind(this)}
+      onSwipingItem={this.onSwipingItem.bind(this)}
```

Comment:
you can just pass these functions directly without binding them to this. Since both of them are arrow functions they  automatically bind `this`.
`this.updateUserLibraryEntry.bind(this)` => `this.updateUserLibraryEntry`
`this.onSwipingItem.bind(this)` => `this.onSwipingItem`